Source/EasyNetQ/RabbitAdvancedBus.cs

```diff
@@ -574,6 +580,12 @@ public virtual void QueuePurge(IQueue queue)
             Preconditions.CheckShortString(name, "name");
             Preconditions.CheckShortString(type, "type");
 
+            // sanity check return the default exchange
+            if (string.IsNullOrEmpty(name))
```

Comment:
and this.... because we check on the `SubscribeToResponse` method. I don't like to change an advanced api just to accommodate an easy check I can do in rpc, this should throw exception if you try to declare the default exchange, that's it.